docker-compose.prod.yml

```diff
@@ -0,0 +1,30 @@
+﻿services:
```

Comment:
Is this and staging really necessary to include? In the end its just a different configuration.

Reply:
To ensure that another server dropout and lost of data, I would prefer to include it in the code base as much as possible.

Reply:
Fair point, agreed